markdown/en/api/ff-paging.api.md

```diff
@@ -3,42 +3,42 @@ ___
 
 
 ### Events
 | Name | Description |
 | ---- | ----------- |
-|**dom-updated**|  This event is triggered when the element has received new data and the template for the element and all sub elements was punched out.|
+| **dom-updated** | This event is triggered when the element has received new data and the template for the element and all sub elements was punched out.|
 
 ## `ff-paging-set`
 ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-|**state** (String) (default:all)| This property sets the condition under which the `ff-paging-set` element will become _aktive_. This is achieved by setting the the numbers behind `currentPage` - for the currently selected page - and `pageCount` - for the amount of pages - through any calculation to the desired value. The condition is then based on this value(e.g. `currentPage > 3 &&pageCount-currentPage >= 3`).|
-|**is-active** (String) **Options**: true, false (default:false)| Determines if the paging set is currently active.|
-|**paging** |(Object) (default:empty) The data from FACT-Finder.|
+| **state** (String) (default: "all") | This property sets the condition under which the `ff-paging-set` element will become _active_. This is achieved by setting the the numbers behind `currentPage` - for the currently selected page - and `pageCount` - for the amount of pages - through any calculation to the desired value. The condition is then based on this value(e.g. `currentPage > 3 &&pageCount-currentPage >= 3`). |
+| **is-active** (Boolean) **Options**: true, false (default: true) | Determines if the paging set is currently active. |
+| **paging** (Object) (default:empty) | The data from FACT-Finder. |
 
 ### Methods
 | Name | Description |
 | ---- | ----------- |
-|**hide()**|Hides the paging set.|
-|**show()**| Shows the paging set.|
+| **hide** | Hides the paging set.|
+| **show** | Shows the paging set.|
 
 ## `ff-paging-item`
 ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-|**type** (String) **Options**: currentLink, firstLink, lastLink, nextLink, previousLink (default:empty) | Via this property you can configure and style different pages and navigation elements of the paging. See dokumentation above.|
-|**products-per-page-item** (Object) (default:empty)|  The paging item data from FACT-Finder.|
-|**show-only** (String) **Options**: true, false (default: false)| Determines that the page is only displayed but not clickable.|
-|**show-selected** (Boolean)| |
+| **type** (String) **Options**: "currentLink", "firstLink", "lastLink", "nextLink", "previousLink" (default:" empty") | Using this property you can configure and style different pages and navigation elements of the paging. See documentation above. |
+| **products-per-page-item** (Object) (default:empty) |  The paging item data from FACT-Finder. |
+| **show-only** (String) **Options**: "true", "false" (default: "")| Determines if pages should only be displayed but not be clickable. |
```

Comment:
(default: ~~""~~ _empty_) That is how all previous not set Strings were described so far

Reply:
That is because all the other Strings had no `value` property. However, this one had a default `value: ""`

Reply:
Please tell me, what I missed - I don't see a `value: ""`
![novalue](https://user-images.githubusercontent.com/42941766/47208572-7665cd00-d38e-11e8-965b-6b60af1f97f4.png)

Reply:
I mistook the property for another one.

But to make sure that the `reflectToAttribute: true` doesn't show an empty string in the HTML-Tag i created a Polymer 1.0 element with the same property.
_Result:_ It doesn't.